src/core/constants.js

```diff
@@ -195,8 +196,17 @@ export const INSTALL_ERROR = 'INSTALL_ERROR';
 
 // Tracking categories.
 export const INSTALL_CATEGORY = 'AMO Addon / Theme Installs';
+export const INSTALL_THEME_CATEGORY = 'AMO Theme Installs';
+export const INSTALL_EXTENSION_CATEGORY = 'AMO Addon Installs';
+
 export const INSTALL_STARTED_CATEGORY = 'AMO Addon / Theme Installs Started';
+export const INSTALL_THEME_STARTED_CATEGORY = 'AMO Theme Installs Started';
+export const INSTALL_EXT_STARTED_CATEGORY = 'AMO Addon Installs Started';
+
 export const UNINSTALL_CATEGORY = 'AMO Addon / Theme Uninstalls';
```

Comment:
Do we still need this? Won't it always be either `UNINSTALL_THEME_CATEGORY` or `UNINSTALL_EXTENSION_CATEGORY`?

Reply:
yep, I liked your suggestion about using *_ACTION instead and will remove these "general" categories which were really being misused :/